src/main/java/com/github/mathematicalguy/moecraft/model/Conlon_model.java

```diff
@@ -0,0 +1,75 @@
+package com.github.mathematicalguy.moecraft.model;
+
+import com.github.mathematicalguy.moecraft.entity.Mrconlon;
+import com.mojang.blaze3d.matrix.MatrixStack;
+import com.mojang.blaze3d.vertex.IVertexBuilder;
+import net.minecraft.client.renderer.entity.model.AgeableModel;
+import net.minecraft.client.renderer.entity.model.EntityModel;
+import net.minecraft.client.renderer.entity.model.VillagerModel;
+import net.minecraft.client.renderer.model.ModelRenderer;
+import net.minecraft.entity.Entity;
+import net.minecraft.util.math.MathHelper;
+
+
+public class Conlon_model<T extends Mrconlon> extends VillagerModel<T> {
```

Comment:
I wouldn't name a class with an`_` in it - it is legal, but weird in java - should probably be `ConlonModel`